src/main/java/com/razorpay/AddonClient.java

```diff
@@ -0,0 +1,20 @@
+package com.razorpay;
+
+public class AddonClient extends ApiClient {
+
+    AddonClient(String auth) {
+        super(auth);
+    }
+
+    /***
+     * To create an Addon, use the createAddon method of SubscriptionClient
+     */
+
+    public Addon fetch(String id) throws RazorpayException {
+        return get(String.format(Constants.ADDON_GET, id), null);
+    }
+
+    public Addon delete(String id) throws RazorpayException {
+        return get(String.format(Constants.ADDON_DELETE, id), null);
```

Comment:
why is delete a get call ?

Reply:
Seems like some mixup in last merge with testing branch ... corrected